src/order/closure.lean

```diff
@@ -113,4 +125,13 @@ def gi : galois_insertion c.to_closed coe :=
   le_l_u := λ x, c.le_closure _,
   choice_eq := λ x hx, le_antisymm (c.le_closure x) hx }
 
+/--
+The galois insertion associated to a closure operator can be used to reconstruct the closure
+operator.
+
+Note that the inverse in the opposite direction does not hold in general.
+-/
+lemma closure_operator_gi_self : closure_operator_of_galois_connection (gi c).gc = c :=
```

Comment:
... then we can go crazy with dot notation:
```suggestion
lemma closure_operator_gi_self : c.gi.gc.closure_operator = c :=
```

Reply:
that's horrible, I love it! Changed, thanks